content/bare/eu/usegalaxy/notices.md

```diff
@@ -1,9 +1,13 @@
-<!--div class="alert" style="background: #FFD500;">
+<div class="alert" style="background: #f8b63e;">
 
-The Freiburg Galaxy team will be striking on 03.03.2023.
-For more information, please see our [blog post](https://galaxyproject.org/news/2023-02-27-climate-strike/).
+#### **Maintenance: Dynamic job router switch**
 
-</div-->
+The European Galaxy server will switch to a new job router, [Total Perspective Vortex](https://total-perspective-vortex.readthedocs.io/).
+It helps us to keep resource management simple and in-sync with other usegalaxy.* instances.
+We have thoroughly tested our new setup. However, we cannot exclude the possibility that there may be temporary disruptions in job routing and that some jobs may not start. We will try to keep this time as short as possible and expect that these restrictions will be over by tomorrow. We apologise for any inconvenience this may cause.
```

Comment:
```suggestion
We have thoroughly tested our new setup. However, we cannot exclude the possibility that there may be temporary disruptions in job routing and that some jobs may not start.
```

Reply:
Looks good to me, but I let you commit those :)